packages/features/ee/billing/teams/internal-team-billing.test.ts

```diff
@@ -0,0 +1,155 @@
+import prismaMock from "../../../../../tests/libs/__mocks__/prismaMock";
+
+import { describe, it, expect, vi, beforeEach, afterEach } from "vitest";
+
+import { purchaseTeamOrOrgSubscription } from "@calcom/features/ee/teams/lib/payments";
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { Redirect } from "@calcom/lib/redirect";
+
+import * as billingModule from "..";
+import { InternalTeamBilling } from "./internal-team-billing";
+
+vi.mock("@calcom/lib/constants", async () => {
+  const actual = await vi.importActual("@calcom/lib/constants");
+  return {
+    ...actual,
+    WEBAPP_URL: "http://localhost:3000",
+  };
+});
+
+vi.mock("..", () => ({
```

Comment:
This just seems weird. Is this the optimal way we should be importing and mocking here?